frontend/src/metabase/dashboard/containers/DashboardApp/DashboardApp.tsx

```diff
@@ -171,24 +188,52 @@ const DashboardApp = (props: DashboardAppProps) => {
     onTimeout,
   });
 
+  const { refreshDashboard } = useRefreshDashboard({
```

Comment:
Those are the hooks that were in DashboardControls

Reply:
It could be that not all of these props are really needed, as some of them may have been needed only in the public/embed ones, for now I just moved them all here.